llama_stack/distribution/store/registry.py

```diff
@@ -144,17 +141,16 @@ async def register(self, obj: RoutableObjectWithProvider) -> bool:
 
 
 async def create_dist_registry(
-    config: StackRunConfig,
+    metadata_store: Optional[KVStoreConfig],
```

Comment:
@dltn please see this refactor. your LlamaStackDirectClient code will need to be updated once this lands.